src/nft/components/NFTCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { useCallback } from 'react';
+import { useAccount } from 'wagmi';
+import { border, cn, color, pressable } from '../../styles/theme';
+import { useIsMounted } from '../../useIsMounted';
+import { useTheme } from '../../useTheme';
+import { LifecycleType, type NFTCardReact } from '../types';
+import { NFTLifecycleProvider } from './NFTLifecycleProvider';
+import { NFTProvider } from './NFTProvider';
+
+export function NFTCard({
+  children,
+  className,
+  contractAddress,
+  tokenId,
+  useNFTData,
+  onStatus,
+  onError,
+  onSuccess,
+}: NFTCardReact) {
+  const componentTheme = useTheme();
+
+  const isMounted = useIsMounted();
+
+  const { chain } = useAccount();
+
+  const handleOnClick = useCallback(() => {
+    const network = chain?.name.toLowerCase() ?? 'base';
```

Comment:
is there somewhere else we can find what `chain` they've configured ? it seems like they would've had to have configured it somewhere right?

Reply:
ahh i see makes sense